analyzers/rspec/cs/S2255_c#.html

```diff
@@ -17,7 +17,8 @@ <h2>Recommended Secure Coding Practices</h2>
 <p>Do not try to encode sensitive information in a non human-readable format before writing them in a cookie. The encoding can be reverted and the
 original information will be exposed.</p>
 <p>Using cookies only for session IDs doesn’t make them secure. Follow <a
-href="https://www.owasp.org/index.php/Session_Management_Cheat_Sheet#Cookies">OWASP best practices</a> when you configure your cookies.</p>
+href="https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#Cookies">OWASP best practices</a> when you configure your
```

Comment:
The link doesn't do what it is meant to do
```suggestion
href="https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#cookies">OWASP best practices</a> when you configure your
```